lib/dogapi/v1/aws_logs.rb

```diff
@@ -0,0 +1,103 @@
+require 'dogapi'
+
+module Dogapi
+  class V1 # for namespacing
+
+    # AwsLogsService for user interaction with AWS configs.
+    class AwsLogsService < Dogapi::APIService
+
+      API_VERSION = 'v1'
+
+      # Get the list of current AWS services for which Datadog offers automatic log collection.
+      # Use returned service IDs with the services parameter for the Enable
+      # an AWS service log collection API endpoint.
+      def aws_logs_list_services
+        request(Net::HTTP::Get, "/api/#{API_VERSION}/integration/aws/logs/services", nil, nil, false)
+      end
+
+      # Create an AWS integration
+      # :config => Hash: integration config.
+      # config = {
+      #   :account_id => '<AWS_ACCOUNT>',
+      #   :lambda_arn => '<LAMBDA_ARN>'
+      # }
+      #
+      # dog = Dogapi::Client.new(api_key, app_key)
+      #
+      # puts dog.add_aws_logs_lambda(config)
+      def add_aws_logs_lambda(config)
+        request(Net::HTTP::Post, "/api/#{API_VERSION}/integration/aws/logs", nil, config, true)
+      end
+
+      # List all Datadog-AWS Logs integrations configured in your Datadog account.
+      def list_aws_logs_integrations
```

Comment:
`list_aws_logs_integrations` or `aws_integrations_logs_list`. In all cases let's have a consistent naming convention everywhere.